tests/lib/node-factory.js

```
@@ -0,0 +1,29 @@
+'use strict'
+
+const createNode = require(`./create-node`)
+const IPFS = require('ipfs')
+
+function NodeFactory () {
+  if (!(this instanceof NodeFactory)) {
+    return new NodeFactory()
+  }
+
+  const nodes = []
+
+  this.createIPFS = async (config, ipfsModule) => {
+    ipfsModule = typeof ipfsModule !== 'undefined' ? ipfsModule : IPFS
+    const node = await createNode(config, ipfsModule)
+    nodes.push(node)
+    return node
+  }
+  this.stopIPFS = () => {
+    nodes.forEach((node) => {
+      node.stop()
+    })
+  }
+  this.getIPFS = (node) => {
```

this should just be a `get()`. Is `node` a positional argument? Maybe we should throw if it's an invalid index.